torchmd/run.py

```diff
@@ -255,8 +261,11 @@ def dynamics(args, mol, system, forces):
                     "ekin": Ekin[k],
                     "etot": Epot[k] + Ekin[k],
                     "T": T[k],
+                    "max_dist": max_dist[k]
                 }
             )
+        if max_dist.max() > args.max_interatomic_dist:
+            break
```

Comment:
Breaking without any kind of message or exception raising seems problematic. How does the user distinguishes between a simulation failing early and a simulation thats finished?